doctr/__main__.py

```diff
@@ -64,6 +64,8 @@ def get_parser():
         help="""Temporary directory used on gh-pages. The default is %(default)r.""")
     deploy_parser.add_argument('--deploy-repo', default=None, help="""Repo to
         deploy the docs to. By default, it deploys to the repo Doctr is run from.""")
+    deploy_parser.add_argument('--no-require-master', dest='nomaster', action='store_true',
```

Comment:
I find it easier to invert the `--no` options (see `--no-upload-key` for example).